shipment.go

```diff
@@ -81,6 +81,8 @@ type ShipmentOptions struct {
 	BillThirdPartyPostalCode string     `json:"bill_third_party_postal_code,omitempty"`
 	ByDrone                  bool       `json:"by_drone,omitempty"`
 	CarbonNeutral            bool       `json:"carbon_neutral,omitempty"`
+	CarrierInsuranceAmount   float64    `json:"carrier_insurance_amount,omitempty"`
```

Comment:
The other `float64` values have 'string' in the json struct tag. I'm pretty sure this is because the EasyPost API will return floats as strings. It's likely we should do it for these new fields as well.